src/main/scala/io/sdkman/changelogs/JExtractMigrations.scala

```diff
@@ -0,0 +1,51 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "082")
+class JExtractMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_jextract_candidate",
+    author = "helpermethod"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "jextract",
+      name = "Jextract",
+      description =
+        "jextract is a tool which mechanically generates Java bindings from a native library headers.",
```

Comment:
I think the official name is `Jextract`. Mind if we change that?

Reply:
Never mind :laughing:

Reply:
I've just copied the description from the GH page 😄